services/ui/public/interfaces/window_tree.mojom

```diff
@@ -320,12 +320,14 @@ interface WindowTree {
   // where the location is (x << 16) | y.
   GetCursorLocationMemory() => (handle<shared_buffer> cursor_buffer);
 
-  // Tells the window manager to start moving the window. OnChangeCompleted is
+  // Tells the (host) window manager to start moving the window. OnChangeCompleted is
   // called on whether the move was canceled. Because there's a delay between
   // when a client sends this message and when the window manager starts acting
-  // on it, pass the cursor location at the start of the move.
+  // on it, pass the cursor location at the start of the move. In external
+  // window move, |drag_offset| is passed to ozone backend for proper handling
```

Comment:
So, what should `drag_offset` be in CrOS? This is a suggestion to offer to the window manager?

Reply:
the drag_offset is the offset for the mouse pointer position, which is used to calculate where a being dragged window should be placed. It is passed to WindowMoveLoopClient, which stores the value and uses it on new mouse move/drag events (on WindowMoveLoopClient::OnMouseMovement) . You can check it here https://github.com/Igalia/chromium/pull/277/commits/4101041e5bf5d5f5052b288a04f53056f308ec51#diff-73743bddabef9dc55d92578838743a5aR31 

ChromeOS does things differently - in case of mus, ToplevelWindowEventHandler::RunMoveLoop is called instead of DesktopWindowTreeHostMus::RunMoveLoop. It doesn't use the drag offset, but instead calls to GetLastTouchPointForTarget or GetLastMouseLocationInRoot to get drag location.

ChromeOs mash does go through DesktopWindowTreeHostMus::RunMoveLoop, but in calls to window_manager with cursor location that translates the cursor to a root window and calculates bounds on drag using that offset.